index.html

```diff
@@ -405,39 +515,75 @@ <h3 id="placement-of-mono-ruby">Placement of mono-ruby</h3>
 <section>
 <h3 id="placement-of-group-ruby">Placement of group-ruby</h3>
 
-<p>Group-ruby is placed as follows:</p>
+<p>
+<span class="checkme">
+In this section, placement rules of group-ruby are described as combinations 
+of two groups of characters, 
+one as 'Western characters' which has proportional width 
```

Comment:
Double quotation marks should be used instead?

Reply:
updated.